test/factories/area.factory.js

```diff
@@ -1,7 +1,7 @@
 import { Factory } from 'rosie';
 
 export default new Factory()
-  .sequence('area', (i) => `Lekki Phase ${i}`)
+  .sequence('area', (i) => `lekki phase ${i}`)
```

Comment:
Can we leave this as the normal case format since we are already converting the saved area and state to lowercase:

```suggestion
  .sequence('area', (i) => `Lekki Phase ${i}`)
```

Reply:
this makes the test fail because the value exists in uppercase but the returned value is in lowercase